mkdocs.yml

```diff
@@ -50,12 +50,15 @@ nav:
         - 'Architecture': 'overview/architecture.md'
         - 'Storage Model': 'overview/storage.md'
         - 'Transaction Model': 'overview/transaction.md'
+        - 'Data Integrity': 'overview/data_integrity.md'
         - 'Fault Model': 'overview/fault.md'
         - 'Security Model': 'overview/security.md'
+        - 'Use Cases': 'overview/use_cases.md'
         - 'Terminology': 'overview/terminology.md'
     - Installation and Setup:
         - 'CentOS': 'QSG/setup.md'
         - 'openSUSE': 'QSG/suse_setup.md'
+        - 'Build from Scratch': 'QSG/scratch.md'
```

Comment:
This is really duplicating what we have in the admin guide ... anyway, let's fix this separately.